meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
```

Comment:
Maybe rephrase "A machine specific bbappend would provide a script and append to the
HOSTAPP_HOOKS variable the name of it"

Reply:
Fixed. Thanks.